tests/countries/test_mauritius.py

```diff
@@ -0,0 +1,222 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+
+from unittest import TestCase
+
+from holidays.countries import Mauritius, MU, MUS
+from tests.common import CommonCountryTests
+
+
+class TestMauritius(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        years = range(1988, 2050)
+        super().setUpClass(Mauritius, years=years)
+        cls.no_estimated_holidays = Mauritius(years=years, islamic_show_estimated=False)
+
+    def test_country_aliases(self):
+        self.assertAliases(Mauritius, MU, MUS)
+
+    def test_no_holidays(self):
+        self.assertNoHolidays(Mauritius(years=1987))
+
+    def test_special_holidays(self):
+        public_holiday = "Public Holiday"
+        for dt, name in (
+            ("2019-07-29", public_holiday),
+            ("2019-09-09", public_holiday),
+        ):
+            self.assertHolidayName(name, dt)
```

Comment:
Considering that we only got two at the moment, what do you think? 👀 @KJhellico @arkid15r 
```suggestion
        self.assertHolidayName("Public Holiday", "2019-07-29", "2019-09-09")
```